packages/file/src/options/directory-list.options.ts

```diff
@@ -0,0 +1,68 @@
+import {MatchTypeEnum} from "../enums/match-type.enum";
+import {TypesEnum} from "../enums/types.enum";
+import {FileInfoInterface} from "../interfaces/file-info.interface";
+import {DirectoryListResultEnum} from "../enums/directory-list-result.enum";
+
+export interface DirectoryListOptions {
+    matchType?: MatchTypeEnum;
```

Comment:
Missing comment here, we should specify the difference between matchType and types